### PR TITLE
feat: support Wagtail v2.15

### LINKED ISSUE #196: Support Wagtail v2.15
https://docs.wagtail.io/en/stable/releases/2.15.html

Allow the latest version of Django and Wagtail in the [install_requires](https://github.com/GrappleGQL/wagtail-grapple/blob/main/setup.cfg#L40) of setup.cfg.

### PR DESCRIPTION
**Pull request by ruisaraiva19**
- Closes #196

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,9 +199,9 @@ Contributions are what make the open source community such an amazing place to b
 
 Wagtail Grapple supports:
 
-- Django 2.2.x, 3.0.x
-- Python 3.6, 3.7, 3.8 and 3.9
-- Wagtail >= 2.5, < 2.13
+- Django 2.2.x, 3.0.x, 3.1.x
```
Comment (zerolab) [inline README.md:202]: suggestion:
```suggestion
- Django 2.2.x, 3.0.x, 3.1.x, 3.2.x
```

**Comment by zerolab** [comment]
@ruisaraiva19 aside from the one minor suggestion, LGTM.
Love the clean up ❤️

Aside: rtd is failing (https://readthedocs.org/projects/wagtail-grapple/builds/15189849/), but not because of this PR.

File: README.md [hunk 2]
```diff
@@ -82,8 +82,8 @@ Getting Grapple installed is designed to be as simple as possible!
 ### Prerequisites
 
 ```
-Django  >= 2.2, <3.1 (<2.3 if using channels)
-Wagtail >= 2.5, <2.13
+Django  >= 2.2 (<2.3 if using channels)
```
Comment (zerolab) [inline README.md:85]: No longer the case since #184

```suggestion
Django  >= 2.2
```

File: README.md (hunk 1 above)
Comment (ruisaraiva19) [reply README.md]: Updated!

File: README.md (hunk 2 above)
Comment (ruisaraiva19) [reply README.md]: Updated!

**Review by zerolab — APPROVED** [review]

**Comment by zerolab** [comment]
Thank you for this, @ruisaraiva19. Merged

**Comment by ruisaraiva19** [comment]
Now we can start preparing a release 😉

**Pull request merged**